app/src/androidTest/assets/features/test.feature

```diff
@@ -1,28 +1,25 @@
 Feature: Login
-  Perform login on email and password are inputted
+  Perform login when a password is entered
 
-  Scenario Outline: Input email and password in wrong format
+  Scenario Outline: Input and validate password next activity
```

Comment:
Again, the Scenario Outline should be a well-structured English sentence, e.g. "Input correct passwords and go to the main activity"